src/test/java/com/courtcanva/ccfranchise/repositories/OrderRepositoryTest.java

```diff
@@ -29,9 +33,13 @@ class OrderRepositoryTest {
     @Autowired
     private OrderRepository orderRepository;
 
+
     @BeforeEach
     void setOrderRepositoryUp() {
         orderRepository.deleteAll();
+        franchiseeRepository.deleteAll();
+        orderRepository.save(OrderTestHelper.Order1());
+        orderRepository.save(OrderTestHelper.Order2());
```

Comment:
I suggest put `orderRepository.save` method to your test method, cause if put in here, it also impact other test method.

Reply:
looks like it's not resoved. `orderRepository.save` si still in `setOrderRepositoryUp()`